secure_collaborative_computation_service/dp_visualization/docs/README.md

```diff
@@ -0,0 +1,52 @@
+# DP-Viz
+
+## 1-Component Overview:
+**Description**:
+The component aims to help data analysts with assessing the privacy-utility trade-off inherent to anonymization and provide visual feedback. Specifically, focusing on anonymization satisfying differential privacy (DP).
+It is a web frontend reachable via the Browser.
+
+Informally, DP requires function-dependent parameterized randomization - provided by so-called *DP mechanisms* typically in the form of additive noise - that hide the impact that a single record can have on a function evaluation.
+For detailed description of differential privacy, we refer to Deliverable D4.1.
+
+**Objectives**:
+The objectives are to provide visual intuition, utility-privacy feedback, and initial parameterization guidance for anonymization with differential privacy.
+
+## 2-Service Dependencies:
+**External Dependencies:**
+- The component relies on the Java version of [Google's DP library](https://github.com/google/differential-privacy) as a backend.
+- SAPUI5 (frontend toolkit for web-based graphical interfaces with HTML/CSS/Javascript)
+- Plotly.js (visualization/graphs)
+- Node.js
+- SpringBoot
+
+**Internal Dependencies:**
+None.
+
+## 3-Service Architecture:
```

Comment:
It is not clear where these components will be deployed, are they will be deployed on the Glaciation cluster or on the another cluster that has Carbyne stack up and running?

Reply:
DP-Viz will be deployed outside of Glaciation. It is a stand-alone component (not related to Carbyne Stack / Computation Service).